README.md

```diff
@@ -207,7 +207,7 @@ A few of these are available in the editor context menu as an experimental featu
 
 ### _Optional_: Debugging
 
-To use the debugger, you must currently manually install `delve`.  See the [Installation Instructions](https://github.com/derekparker/delve/tree/master/Documentation/installation) for full details.  On MacOS it requires creating a self-signed cert to sign the `dlv` binary.
+To use the debugger, you must currently manually install `delve`.  See the [Installation Instructions](https://github.com/derekparker/delve/tree/master/Documentation/installation) for full details.  On MacOS it requires creating a self-signed cert to sign the `dlv` binary.  If using WSL on Windows, you will need the WSL 2 Linux kernel.  See [WSL 2 Installation](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) and note the Window 10 build version requirements. 
```

Comment:
> and note the Window 10 build version requirements.

Its not clear to me as to why we want to call this out. Can you elaborate?

Reply:
The default WSL uses syscall mapping, hence delve just hangs on the trace() call.   You need to join the Windows Insiders program and get the latest version of the OS, and then you can install the WSL 2 kernel, which is a true virtualized machine.